source/initial_conditions/S40RTS_perturbation.cc

```diff
@@ -625,6 +631,10 @@ namespace aspect
                              Patterns::Double (0),
                              "The reference temperature that is perturbed by the spherical "
                              "harmonic functions. Only used in incompressible models.");
+          prm.declare_entry ("Remove temperature heterogeneity down to specified depth", "-10",
+                             Patterns::Double (),
+                             "This will set the heterogeneity prescribed by S20RTS or S40RTS to zero "
+                             "down to the specified depth.");
```

Comment:
I would like to set it to zero, but didn't because I thought this might cause a problem if the depth of a given point isn't exactly 0 but slightly smaller (due to numerical error). Can that not happen? The value of -10 is random, could also be a very large negative value.